### PR TITLE
[FIX] l10n_be: Account 451200 must be reconciliable

### PR DESCRIPTION
**Pull request by Julien00859**
opw-1913108
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_be/data/account_pcmn_belgium_data.xml [hunk 1]
```diff
@@ -558,6 +558,7 @@
         <record id="a4512" model="account.account.template">
             <field name="name">Compte courant administration T.V.A.</field>
             <field name="code">4512</field>
+            <field name="reconcile" eval="True"></field>
```
Comment (nim-odoo) [inline addons/l10n_be/data/account_pcmn_belgium_data.xml]: It could use a self-closing tag:
```
<field name="reconcile" eval="True"/>
```

**Comment by Julien00859** [comment]
@nim-odoo done

**Comment by Julien00859** [comment]
robodoo retry

**Comment by nim-odoo** [comment]
robodoo retry

**Pull request closed without merging**